service/history/timerQueueActiveTaskExecutor.go

```diff
@@ -664,3 +677,7 @@ func (t *timerQueueActiveTaskExecutor) emitTimeoutMetricScopeWithNamespaceTag(
 		metricsScope.IncCounter(metrics.HeartbeatTimeoutCounter)
 	}
 }
+
+func (t *timerQueueActiveTaskExecutor) addDefaultTimeout(ctx context.Context) (context.Context, context.CancelFunc) {
+	return context.WithTimeout(ctx, taskTimeout)
+}
```

Comment:
is this wrapper necessary?

Reply:
other than this file, other part of the PR use `ctx, cancel = context.WithTimeout(ctx, taskTimeout)`